SalesforceFoundation/k12-architecture-kit
# [K12] Picklist Updates to Case(Behavior Involvement) and Behavior Response Objects

# Critical Changes

# Changes
## New Behavior Response-Related Picklist Values
We've added new picklist values to the existing "Type" field on Behavior Response. 
- Detention
- Expulsion with Services
- In-School Suspension
- Out-of-School Suspension
- Removal from Classroom

We've also added these picklist values to the existing "Location" field on Case for Behavior Response. 
- Administrative offices area
- Athletic field or playground
- Auditorium
- Bus stop
- Cafeteria area
- Classroom
- Computer lab
- Hallway or stairs
-  Library/media center
- Locker room or gym areas
- Online
- Parking lot
- Restroom
- School bus
- Stadium
- Unknown
- Walking to or from school

For new installations, the new picklist values are automatically included. If you installed an earlier EDA release, manually add the new picklist values to the "Type" field on Behavior Response and to the "Location" field on Case for Behavior Response as indicated above. For information, check out [Add Custom Fields and Picklist Values](https://powerofus.force.com/s/article/EDA-Add-Custom-Picklist-Values).

# Issues Closed
Closes #198
# New Metadata
New picklist values for hed__Type__c on Behavior Response: 
- Detention
- Expulsion with Services
- In-School Suspension
- Out-of-School Suspension
- Removal from Classroom

New picklist values for hed__Location__c on Case: 
- Administrative offices area
- Athletic field or playground
- Auditorium
- Bus stop
- Cafeteria area
- Classroom
- Computer lab
- Hallway or stairs
-  Library/media center
- Locker room or gym areas
- Online
- Parking lot
- Restroom
- School bus
- Stadium
- Unknown
- Walking to or from school

# Deleted Metadata

# Testing Notes
Dev org: 
- Create a dev org and run dev_org flow and check that the new picklist values are showing up alphabetically and that "Other" is set at the end of the list. For Hed__Location__c, checking the picklist values via Incident record type, the "Other" is not set at the end of the list. 

Trial org: 
- Create a trial org and run trial_org flow and check that the new picklist values are showing up alphabetically and that "Other" is set at the end of the list. For Hed__Location__c, checking the picklist values via Incident record type, the "Other" is not set at the end of the list. 

- Create a trial org and run net_new_org flow and check that the new picklist values are showing up alphabetically and that "Other" is set at the end of the list. For Hed__Location__c, checking the picklist values via Incident record type, the "Other" is not set at the end of the list.
